apps/test/unit/lib/ui/SettingsCogTest.js

```diff
@@ -99,29 +106,17 @@ describe('SettingsCog', () => {
         assets.showAssetManager.restore();
       });
 
-      it('is the first menu item', () => {
```

Comment:
I was having a really hard time transitioning it to Enzyme 3. Then I started thinking that perhaps there is a reason there is no easy way to unit test this particular thing. I thought about the test itself and it seemed like a bit of an odd thing to unit test, since it was a visual property that doesn't depend on state or logic within the component. The equivalent that I could think of was testing whether a button's color is red when the button never changes. Perhaps it's something that could be covered in an eyes test, but didn't seem like a unit test. 

I'd be interested in hearing your thoughts :)

Reply:
I follow your thinking for sure. An eyes or cucumber test does seem the best place for this type of check. Do you know if we have current coverage of this?

Reply:
Talked offline and we decided that this isn't a necessary test.